projects/packages/videopress/src/client/admin/components/video-filter/index.tsx

```diff
@@ -51,11 +53,22 @@ export const FilterButton = ( props: {
 	);
 };
 
+const DisabledReasonTooltip = ( props: { message: string } ): JSX.Element => {
+	return (
+		<Tooltip position="middle center" text={ props.message }>
+			<span className={ classnames( styles[ 'title-adornment' ] ) }>
```

Comment:
We don't need to wrap class by `classnames()` function here:

```suggestion
			<span className={ styles[ 'title-adornment' ] }>
```